vaadin-button-flow-parent/vaadin-button-flow/src/main/java/com/vaadin/flow/component/button/Button.java

```diff
@@ -340,6 +342,7 @@ public void setDisableOnClick(boolean disableOnClick) {
         this.disableOnClick = disableOnClick;
         if (disableOnClick) {
             getElement().setAttribute("disableOnClick", "true");
```

Comment:
Could change this undefined attribute to a property.

Reply:
Please don't ;) I'm relying on this attribute for styling

Reply:
Oh, ok, and you're probably not the only one :) Let's keep it as an attribute then.

...maybe we can align it with our other component attributes and change it to `<vaadin-button disable-on-click>` in one of the upcoming majors.